mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping

```diff
@@ -50,7 +50,8 @@ CLASS net/minecraft/class_2609 net/minecraft/block/entity/AbstractFurnaceBlockEn
 	METHOD method_17763 dropExperienceForRecipesUsed (Lnet/minecraft/class_3222;)V
 		ARG 1 player
 	METHOD method_26395 isNonFlammableWood (Lnet/minecraft/class_1792;)Z
-		COMMENT Returns whether the provided {@code item} is in the {@linkplain ItemTags#NON_FLAMMABLE_WOOD} tag.
+		COMMENT Returns whether the provided {@code item} is in the {@linkplain
+		COMMENT net.minecraft.tag.ItemTags#NON_FLAMMABLE_WOOD non_flammable_wood} tag.
```

Comment:
```suggestion
		COMMENT Returns whether the provided {@code item} is in the {@link
		COMMENT net.minecraft.tag.ItemTags#NON_FLAMMABLE_WOOD non_flammable_wood} tag.
```
I'd say a non-plaintext link is better here.

Reply:
Hmm, seems link is like wrapping stuff in `<code>`, so this actually works better I guess?

Reply:
Yeah `{@link}` makes it use the code font which is better for a tag.